src/integrations/intercom/serializers/intercom-attributes.js

```diff
@@ -3,6 +3,17 @@ const JSONAPISerializer = require('jsonapi-serializer').Serializer;
 function serializeIntercomAttributes(attributes, collectionName, meta) {
   const type = `${collectionName}_intercom_attributes`;
 
+  const unixTimestampToDateOrNull = (unixTimestamp) =>
+    unixTimestamp && new Date(unixTimestamp * 1000);
+
+  // Attributes keys ending with `_at` are unix timestamp
+  // thus they need to be converted to js Date
+  Object.entries(attributes).forEach(([attributeKey, attributeValue]) => {
+    if (attributeKey.endsWith('_at')) {
+      attributes[attributeKey] = unixTimestampToDateOrNull(attributeValue);
+    }
+  });
+
```

Comment:
No need to use `new Date(...)`, as only the timestamp can do the job.
Works well this way :)

Reply:
I agree, but looking at other dates sent by the agents, it seems like they are sent as `new Date()` instead of timestamp, so I'll keep it this way for consistency